x-pack/plugins/apm/common/environment_filter_values.ts

```diff
@@ -37,6 +37,18 @@ export function getEnvironmentLabel(environment: string) {
   return environmentLabels[environment] || environment;
 }
 
+export function omitEsFieldValue({
+  esFieldValue,
+  value,
+  text,
+}: {
+  esFieldValue?: string;
+  value: string;
+  text: string;
+}) {
+  return { value, text };
+}
```

Comment:
This is needed due to a console error introduced in https://github.com/elastic/kibana/pull/95903#discussion_r642516545

I don't think the console error in itself is an issue but it made it really hard to make the changes needed for this PR when the console was full of unrelated errors.